src/pages/users/[userId]/requests.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { NextPage } from 'next';
+import RequestList from '../../../components/RequestList';
+import useRouteGuard from '../../../hooks/useRouteGuard';
+import { Permission } from '../../../hooks/useUser';
+
+const UserRequestsPage: NextPage = () => {
+  useRouteGuard(Permission.MANAGE_REQUESTS);
```

Comment:
This guard will prevent people who can view requests from access this also?

Reply:
Ah right, will add the View Requests perm as well.  Good catch, thanks 😄